api/v1/configurationpolicy_types.go

```diff
@@ -27,7 +27,8 @@ type RemediationAction string
 // +kubebuilder:validation:Enum=low;Low;medium;Medium;high;High;critical;Critical
 type Severity string
 
-// PruneObjectBehavior : DeleteAll, DeleteIfCreated, or None
+// PruneObjectBehavior is used to clean up resources that are created by the
+// policy. Values: DeleteAll, DeleteIfCreated, or None (default)
```

Comment:
I think the values come up nicely with `kubectl explain` since the enum is set.
```suggestion
// policy upon policy deletion.
```